projects/pgai/pgai/vectorizer/worker_tracking/worker_tracking.py

```diff
@@ -56,6 +57,10 @@ async def force_heartbeat(self) -> None:
         if not self.enabled:
             return
 
+        if self.heartbeat_task is not None:
```

Comment:
NIT: If the intention of `force_heartbeat` is also to stop the background task, then I'd name it `force_heartbeat_and_stop`. Feel free to leave as is.